cas-client-core/src/main/java/org/jasig/cas/client/configuration/ConfigurationKeys.java

```diff
@@ -54,7 +54,6 @@
     ConfigurationKey<String> CAS_SERVER_URL_PREFIX = new ConfigurationKey<String>("casServerUrlPrefix", null);
     ConfigurationKey<String> ENCODING = new ConfigurationKey<String>("encoding", null);
     ConfigurationKey<Long> TOLERANCE = new ConfigurationKey<Long>("tolerance", 1000L);
-    ConfigurationKey<Boolean> DISABLE_XML_SCHEMA_VALIDATION = new ConfigurationKey<Boolean>("disableXmlSchemaValidation", Boolean.FALSE);
```

Comment:
we may need to keep this but make it a no-op for compatibility (otherwise I believe the not reserved params get appended to queries)